pkg/provider/operators.go

```diff
@@ -318,3 +319,8 @@ func getOperatorTargetNamespaces(namespace string) ([]string, error) {
 
 	return list.Items[0].Spec.TargetNamespaces, nil
 }
+
+func IsValidSemanticVersion(version string) bool {
```

Comment:
I think this function would fit better into some generic "utils" package as the semantic versioning check applies to any software version, not just to Operators.

Reply:
I would consider that as part of refactoring once we have some more operator related issues implemented.  It's too early to refactor now.